src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawFigure {
+    void drawFigure();
```

Comment:
```suggestion
public interface Drawable {
    void draw();
```

Мы можем использовать этот интерфейс не только для того, что б задавать поведение фигурам, но также и другим объектам, которые можно нарисовать(картины, графики и тд...). Давай обобщим его название